app/gist/controller.js

```diff
@@ -22,14 +29,32 @@ export default Ember.Controller.extend({
    * @type {String}
    */
   buildOutput: '',
+
+  /**
+   * If the code is currently being built
+   * @type {boolean}
+   */
   isBuilding: false,
+
+  /**
+   * If the edited code has not been saved by the user
+   * @type {boolean}
+   */
   unsaved: true,
+
+  /**
+   * File in the current active editor column
+   * @type {Object}
+   */
   activeFile: null,
+
+  /**
+   * Column which has the currently focused editor
+   * @type {Number}
+   */
   activeEditorCol: null,
-  col1File: null,
-  col2File: null,
-  col1Active: equal('activeEditorCol','1'),
-  col2Active: equal('activeEditorCol','2'),
+
+  columns: [],
```

Comment:
should likely not put a complex object on the prototype, especially if it is meant to hold instance state.